src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -191,14 +189,10 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
   private stepChangeBlockedByCaptcha = false;
   @Input({ required: true }) donor: Person | undefined;
 
-  public tipControlStyle: 'dropdown'|'slider';
-
-  private tipAmountFromSlider: number;
+  public tipControlStyle: 'dropdown';
```

Comment:
My understanding is that if we trial a slider now, it will likely be using an off-the-shelf component with better accessibility. So rather than figures out its dependencies, use of NgZone etc., I removed the custom slider for now.